test/tests/minimal_app/tests

```diff
@@ -6,7 +6,7 @@
     # Make sure --minimal flag is working
     type = RunApp
     cli_args = '--minimal --list-constructed-objects'
-    expect_out = "FEProblem\nDefaultNonlinearConvergence\nGeneratedMesh"
+    expect_out = "FEProblem\n*GeneratedMesh"
```

Comment:
You could just move the convergence object ahead of the problem

Reply:
I saw someone on discussion get a test failure there. I think this test is brittle

Reply:
Yes it's because @joshuahansel must have done a `git sed` to change the name and then didn't change the order here